examples/deno-router/components/App.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, register, xml, Router, Route, HashRouter } from "../src/mod.ts";
+// has router
+class AppRoot extends Component {
+    override template = xml`
+      <${Router}>
+        <${Route} prop:path=${"/"} prop:content=${"/components/landing.js"}></${Route}>

```

Comment:
so, the issue with just taking a string here is that it's not amenable to bundling; doing something like:
```ts
xml`<${Route} prop:content=${() => import('/components/landing')}>`
```
would make it statically analyzable.

Reply:
But that will increase the bundle size no? Because all the code from those files will be part of the bundle

Reply:
Not if you leave it split; I've seen a lot of things like:

main.js (contains the initialization code and some dynamic imports)
modules/module_1.js (contains the bundle for a single page incl. all components that it uses)
modules/module_2.js (contains the bundle for a single page incl. all components that it uses)

and then some commonly used components might be chunked into a separate chunk used by both module_1 or module_2.

The goal here is to reduce the total number of files that need to be traversed to figure out the full dependency graph; since otherwise, you'd need to load A -> B -> C -> D -> E -> etc. for however many levels of nesting there are. Doing _some_ bundling reduces the number of round trips necessary.